### PR TITLE
PEP-8 fixes

### PR DESCRIPTION
**Pull request by martinpitt**
Mostly done with autopep8.
    
Keep a few remaining ignores which are hard/impractical to fix.

### CODE REVIEW

**Comment by martinpitt** [comment]
The remaining test failures are unrelated. The cockpit selenium tests are currently rather impossible to get green..

File: tests-score [hunk 1]
```diff
@@ -36,13 +36,17 @@ BOTS = os.path.dirname(os.path.realpath(__file__))
 # https://github.com/cockpit-project/cockpituous/blob/master/learn/README.md
 
 # Here we're looking for a field in a record that only has one value
+
```
Comment (marusak) [inline tests-score]: here it would make more sense to put those two newlines above the function description.

File: tests-score [hunk 2]
```diff
@@ -36,13 +36,17 @@ BOTS = os.path.dirname(os.path.realpath(__file__))
...
 def value(record, field):
     values = record.get(field, [])
     if len(values) == 1:
         return values[0][0] or ""
     return None
 
 # Here we're looking for the count of a specific field/value in the record
+
+
```
Comment (marusak) [inline tests-score]: detto

File: tests-scan [hunk 3]
```diff
@@ -123,13 +126,16 @@ def tests_human(priority, name, number, revision, ref, context, base, original_b
...
     )
 
+
 def is_internal_context(context):
     for pattern in ["rhel", "edge", "vmware", "openstack"]:
         if pattern in context:
             return True
     return False
 
 # Prepare a test invocation command
```
Comment (marusak) [inline tests-scan]: I would keep function description right next to the function itself and put those two newlines above the description.

File: tests-scan [hunk 4]
```diff
@@ -106,6 +107,8 @@ def main():
     return 0
 
 # Prepare a human readable output
+
```
Comment (marusak) [inline tests-scan]: detto

File: tests-score (hunk 1 above)
Comment (martinpitt) [reply tests-score]: Well spotted, thanks! Indeed autopep doesn't handle these well. I went through the entire diff and fixed all of these.

**Comment by martinpitt** [comment]
@marusak: The interdiff is fairly trivial, so I triggered just two tests. Is that ok for you?

**Comment by martinpitt** [comment]
Argh, [host test failure](https://209.132.184.41:8493/logs/pull-33-20190927-085757-890b2d5b-cockpit-project-bots--host/log) -- that was supposed to be fixed by PR #31, but is apparently still the case.

**Pull request merged**